### ISSUE TITLE
Do not open link using right mouse button

### ISSUE THREAD

**Issue by fadeevsg** [tabkit/tabkit2#116]
If you click right mouse button on link and choose "Open Link in New Tab", then new tab will not open. FF 47.0, Tab Kit 2nd Edition 0.12.14, Windows XP SP3.

**Comment by PikachuEXE** [comment]
Does it work on FF 45?
You can download a portable version of it at http://portableapps.com/apps/internet/firefox-portable-esr

**Comment by M3Rocket** [comment]
Works in 46.0.1. Haven't tried 47.0 yet.

**Comment by fadeevsg** [comment]
> Does it work on FF 45?

Yes. It works. I downloaded portable version 45.1.1 ESR and tried extension on it.

Tab Kit 2nd Edition worked fine on version 46.0.1. It was not work properly immediately after upgrading to version 47.0.

**Comment by jjeziorny** [comment]
It seems broken with FF 47, I can't even click on links with the LEFT mouse button, after disabling all my addons and enabling one by one foudn the problem was with tabkit2 :(  it's the one I most need. buaaaaaa

**Comment by aernhart** [comment]
Issue with Right mouse button "Open Link in New Tab" appeared after upgrade to FF 47 (Fedora Linux). In new FF profile, with no Tablit2 installed, this works as expected, but profiles with Tabkit2 it fails. Left mouse button works as expected, unlike experience of jjeziorny.
I looked briefly into FF dev console, etc. to look for any useful details to offer, but didn't turn up anything. Do let me know if there's any investigation or testing I could offer. Thx much!

**Comment by mattehicks** [comment]
Bumping here, Firefox V47 and tabkit 2nd ed. 0.12.15 
Still not working; Right-click and 'Open link in new tab'

(love the addon by the way!)

**Comment by desirepl** [comment]
FF47 and now FF48 - same problem. Not working and found that this bug is connected to TabKit2

**Comment by LookingForWayOut** [comment]
I finally came here, after experiencing the issue for several weeks. Putting my name on this list.
TK2 0.12.15.
FF 48.0.2.

**Comment by PDunbar1** [comment]
Pardon duplicate on Gitter.  Just finding this thread and subscribing.  Same problem also found on FF49.
